_posts/2021-05-05-Pivoting.md

```diff
@@ -38,7 +38,7 @@ addressed:
 - Objects should have a proper owner reference chain.
 
 For a detailed explanation of the above-mentioned prerequisites please read the
-[pivoting documentation](https://github.com/metal3-io/metal3-docs/blob/master/docs/move.md).
+[pivoting documentation](https://github.com/metal3-io/metal3-docs/blob/main/docs/user-guide/src/capm3/pivoting.md).
```

Comment:
Anytime content is pointing to user-guide, then we should link the rendered output at metal3-io. In this case that would be https://book.metal3.io/capm3/pivoting

Reply:
Makes sense, updated